todo_app/templates/index.html

```diff
@@ -72,6 +76,7 @@ <h5 class="text-center">✅COMPLETED✅</h5>
     <li class="list-group-item">
         <h5>
         {{card.name}}
+        {% if is_admin %}
```

Comment:
Nice - it would be a bonus if this also applied to the "new task" form too (since currently I can see it as a non-admin but can't use it) but it's admittedly quite helpful for testing as is!